options.go

```diff
@@ -130,11 +130,12 @@ type Options struct {
 	// Set Verbose to true to enable more text logging.
 	Verbose bool `yaml:"verbose"`
 
-	// DEPRECATED: set `UseThrift` to true if you do not want gRPC
-	UseGRPC bool `yaml:"usegrpc"`
-
-	// Switch to
+	// Force the use of a specific transport protocol.
+	// If multiple are set to true, the top one is chosen.
+	// If none are set to true, GRPC is defaulted to.
 	UseThrift bool `yaml:"use_thrift"`
+	UseHttp bool `yaml:"use_http"`
+	UseGRPC bool `yaml:"usegrpc"`
```

Comment:
NIT: is there a reason this isn't `use_grpc` for consistency?

Reply:
I am just copying an already existing field. Changing the yaml would be backward incompatible and mean I would need to do a major instead of a minor release. I would like to avoid that.